exla/lib/exla/lib.ex

```diff
@@ -123,31 +123,31 @@ defmodule EXLA.Lib do
   @doc """
   Returns a minimum value scalar operator for the given type.
 
-  Minimum values are defined in `Nx.Type.min_value_binary/1`.
+  Minimum values are defined in `Nx.Type.min_finite_binary/1`.
   """
-  def min_value(%Builder{} = builder, type) do
-    Op.constant_from_binary(builder, min_value_binary(type), Shape.make_shape(type, {}))
+  def min_finite(%Builder{} = builder, type) do
+    Op.constant_from_binary(builder, min_finite_binary(type), Shape.make_shape(type, {}))
   end
 
   @doc """
   Returns a maximum value scalar operator for the given type.
 
-  Maximum values are defined in `Nx.Type.max_value_binary/1`.
+  Maximum values are defined in `Nx.Type.max_finite_binary/1`.
   """
-  def max_value(builder, type) do
-    Op.constant_from_binary(builder, max_value_binary(type), Shape.make_shape(type, {}))
+  def max_finite(builder, type) do
+    Op.constant_from_binary(builder, max_finite_binary(type), Shape.make_shape(type, {}))
   end
 
   defp subbuilder(%Builder{name: name} = builder, desc) do
     suffix = System.unique_integer([:positive])
     Builder.new(builder, name <> "-" <> desc <> "-" <> Integer.to_string(suffix))
   end
 
-  defp min_value_binary({:pred, 8}), do: <<0>>
-  defp min_value_binary(type), do: Nx.Type.min_value_binary(type)
+  defp min_finite_binary({:pred, 8}), do: <<0>>
+  defp min_finite_binary(type), do: Nx.Type.min_finite_binary(type)
 
-  defp max_value_binary({:pred, 8}), do: <<1>>
-  defp max_value_binary(type), do: Nx.Type.max_value_binary(type)
+  defp max_finite_binary({:pred, 8}), do: <<1>>
+  defp max_finite_binary(type), do: Nx.Type.max_finite_binary(type)
```

Comment:
@polvalente @seanmor5 I would like your thoughts on this. We have a `Nx.Type.max_finite_binary(type)` that returns the underlying binary, not a tensor. `Nx.Constants.max_finite(type)` is simply a wrapper around the `Nx.Type` function.

We could remove the `Nx.Type` function and instead do something like this here:

    Nx.Constants.max_finite(type, backend: Nx.BinaryBackend).data.binary

So the question is: do we want to keep the `Nx.Type.*_binary` functions for low-level access? Or should we remove them?

Reply:
I think that keeping things under `*_binary` allows for future `tensor.data` changes, where we don't necessarily need to keep the representation under `tensor.data.binary`. Other than that, removing those would mean a less bloated API.

Is the `Nx.Type` module public? If so, I would remove the `*_binary` functions and use the `.data.binary` syntax instead. Otherwise, I don't have a clear preference on that